view/template/app-user-guide.hbs

```diff
@@ -0,0 +1,286 @@
+<main class="container bear-background">
+    <h1 class="text-xs-center">
+        PROMIS - User Guide For The Mobile App
+    </h1>
+    <h2>
+        Please follow below mentioned steps to download the app on android device and start the survey
+    </h2>
+    <br>
+    <br>
+    <nav>
+        <h2>
+            Index:
+        </h2>
+        <ol>
+            <li>
+                <a href="#step1">
+                    Download the app on mobile device
+                </a>
+            </li>
+            <li>
+                <a href="#step2">
+                    After download, install the app
+                </a>
+            </li>
+            <li>
+                <a href="#step3">
+                    If no notification displayed, go to Downloads to install
+                </a>
+            </li>
+            <li>
+                <a href="#step4">
+                    Installation fails due to unknown source
+                </a>
+            </li>
+            <li>
+                <a href="#step5">
+                    Click Install
+                </a>
+            </li>
+            <li>
+                <a href="#step6">
+                    Open the app
+                </a>
+            </li>
+            <li>
+                <a href="#step7">
+                    Go to Settings page of app
+                </a>
+            </li>
+            <li>
+                <a href="#step8">
+                    Enter PIN and Server Address
+                </a>
+            </li>
+            <li>
+                <a href="#step9">
+                    Ready to take the Survey
+                </a>
+            </li>
+            <li>
+                <a href="#step10">
+                    For Issues - Contact Us
+                </a>
+            </li>
+        </ol>
+    </nav>
+    <br>
+    <br>
+    <article>
+        <section id="step1">
+            <h2>
+                Step 1: Download the app on mobile device
+            </h2>
+            <h3>
+                Enter the below URL in the web browser of your mobile device and press Enter
+            </h3>
+            <h3>
+                URL:
+                <a href="https://swent1linux.asu.edu:8085/PROMIS">
+                    https://swent1linux.asu.edu:8085/PROMIS
+                </a>
+            </h3>
+            <h3>
+                Screenshot:
+            </h3>
+            <img src="/static/image/image1.png" alt="Step 1" height="600" width="500">
```

Comment:
it seems like many of the images have manual height and width set, and set values are different than the original aspect ratio. (e.g. this is set to 500x660, the original ratio is 1080x1920)
This will cause image warping on most browsers.

We should either only set height or width (just one), or make sure the set ratio of both height and width matches the original.